bindata/network/openshift-sdn/003-rbac-controller.yaml

```diff
@@ -0,0 +1,160 @@
+# Only for testing against vanilla k8s
+# will go away when running against the real installer
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: openshift-infra
+
+---
+# The controller also requires a service account in openshift-infra
+# It will use this manually, regardless of which account is configured in the pod
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: sdn-controller
+  namespace: openshift-infra
+
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: sdn-controller
+  namespace: openshift-sdn
+
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: openshift-sdn-controller
+rules:
+- apiGroups: [""]
+  resources:
+  - namespaces
+  verbs:
+  - list
+  - get
+  - watch
+- apiGroups: ["network.openshift.io"]
+  resources:
+  - clusternetworks
+  - egressnetworkpolicies
+  - hostsubnets
+  - netnamespaces
+  verbs:
+  - get
+  - list
+  - watch
+  - create
+  - update
+  - patch
+- apiGroups: [""]
+  resources:
+  - nodes
+  verbs:
+  - list
+  - get
+  - watch
+
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: openshift-sdn-controller
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: openshift-sdn-controller
+subjects:
+- kind: ServiceAccount
+  name: sdn-controller
+  namespace: openshift-sdn
+
+---
+# bind the openshift-sdn-controller role to the service-account in openshift-infra
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: openshift-sdn-controller-infra
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: openshift-sdn-controller
+subjects:
+- kind: ServiceAccount
+  name: sdn-controller
+  namespace: openshift-infra
+
+---
+# Only grant access to the specific config map for leader election
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: openshift-sdn-controller-leaderelection
+  namespace: kube-system
```

Comment:
The current version of the controller PR uses the openshift-sdn namespace for this